nfsstat/check.py

```diff
@@ -82,8 +82,10 @@ def _parse_ops(self):
     def _parse_read_data(self):
         read_data = self._device_data[4]
         self.read_ops = float(read_data[0])
-        self.read_kb = float(read_data[1])
-        self.read_kb_per_s = float(read_data[2])
+        self.read_kb_per_s = float(read_data[1])
+        self.read_kb = float(read_data[2])
```

Comment:
I think renaming this to `self.read_kb_per_op` would make the check easier to read.